mde-services/src/main/java/de/terrestris/mde/mde_backend/model/json/Service.java

```diff
@@ -65,7 +65,4 @@ public enum ServiceType {
 
   private List<ColumnInfo> columns;
 
-  // TODO: update importer according to excel sheet
-  private List<DownloadInfo> downloads;
```

Comment:
This field is still needed. It contains the information of fields 68-71 and is used in the Service Form. It is different from the ColumnInfo.

Reply:
Hm, I'm unsure about that. Aren't the previews handled by the `preview` fields of the dataset and the services?

Reply:
This is not about previews. A service can contain `n` attributes or `downloads`. The `downloads` field contains this information.

![image](https://github.com/user-attachments/assets/d7645ed0-a8da-4065-9d31-8cea827b9dac)

Reply:
Alright, I re-added it, although I'm currently unsure where to output this during ISO generation (this is only relevant for ATOM download links).